app/controllers/reviews_controller.rb

```diff
@@ -3,14 +3,16 @@ class ReviewsController < ApplicationController
   before_action :find_booking
 
   def new
-    @review = Review.new(review_params)
```

Comment:
Why do we need to remove the strong params here?

Reply:
We did not have it in desk-dash so was wondering why we needed it in local response.